gh_pages/_source/getting_started_doc.rst

```diff
@@ -0,0 +1,67 @@
+Getting Started
+===============
+
+Install ROS and Catkin
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+`Install ROS Noetic <http://wiki.ros.org/noetic/Installation/Ubuntu>`_.
+
+It is easy to miss steps when going through the ROS installation tutorial. If you run into errors in the next few steps, a good place to start is to go back and make sure you have installed ROS correctly.
+
+Once you have ROS installed, make sure you have the most up to date packages: ::
+
+  rosdep update
+  sudo apt-get update
+  sudo apt-get dist-upgrade
+
+Install `catkin <http://wiki.ros.org/catkin>`_ the ROS build system: ::
+
+  sudo apt-get install ros-noetic-catkin python-catkin-tools
+
+Install `wstool <http://wiki.ros.org/wstool>`_ the ROS workspace management tool: ::
+
+  sudo apt-get install python-wstool
+
+Build Instructions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Create a workspace (in this case we'll name it `tesseract_ws`): ::
+
+  mkdir -p ~/tesseract_ws/src
+
+Move into source directory: ::
+
+  cd ~/tesseract_ws/src 
+
+Clone Tesseract Planning repository into your workspace: ::
+
+  git clone https://github.com/ros-industrial-consortium/tesseract_planning
+
+If you are using the Robot Operating System (ROS), you can also clone the Tesseract ROS repository: ::
+
+  git clone https://github.com/ros-industrial-consortium/tesseract_ros
+
+Clone the repositories in the dependencies.rosinstall file using wstool or some other method (e.g. manually git cloning them): ::
+
+  wstool init ~/tesseract_ws/src/ ~/tesseract_ws/src/tesseract_planning/dependencies.rosinstall
```

Comment:
Are these tutorials meant to extend to repositories beyond the core `tesseract` repository? It's not actually necessary to build `tesseract_planning` and `tesseract_ros` in order to build `tesseract`, so we might consider dropping these instructions and putting tutorials for those projects in their respective repositories